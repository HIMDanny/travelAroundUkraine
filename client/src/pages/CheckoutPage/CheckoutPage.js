import React from 'react';
import { Container, Typography } from '@mui/material';
import { CheckoutForm } from '../../features/checkout/components';

const CheckoutPage = () => (
  <Container sx={{ mt: '150px', mb: '150px' }}>
    <Typography variant="h2" align="center" sx={{ mb: '20px' }}>
      Checkout
    </Typography>
    <CheckoutForm />
  </Container>
);

export default CheckoutPage;
