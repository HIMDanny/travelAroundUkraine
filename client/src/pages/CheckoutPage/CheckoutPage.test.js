/* eslint-disable react/prop-types */
import { render } from '@testing-library/react';
import CheckoutPage from './CheckoutPage';

jest.mock('@mui/material/Typography', () => ({ children }) => <p>{children}</p>);
jest.mock('@mui/material/Container', () => ({ children }) => <div>{children}</div>);
jest.mock('../../features/checkout/components/CheckoutForm/CheckoutForm', () => ({ children }) => (
  <div>CHECKOUT FORM</div>
));

describe('CheckoutPage snapshot test', () => {
  test('should CheckoutPage match snapshot', () => {
    const { asFragment } = render(<CheckoutPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
