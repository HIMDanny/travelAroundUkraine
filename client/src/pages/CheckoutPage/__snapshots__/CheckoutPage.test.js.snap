// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutPage snapshot test should CheckoutPage match snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Checkout
    </p>
    <div>
      CHECKOUT FORM
    </div>
  </div>
</DocumentFragment>
`;
